.github/workflows/nim_test.yml

```
@@ -20,7 +20,7 @@ jobs:
           - macos-latest
           - windows-latest
         nim-version:
-          - '2.0.8'
+          - '2.2.0'
```

```suggestion
          - '2.X'
```
https://github.com/jiro4989/setup-nim-action#setup-a-latest-minor-version-nim

---

Tried but doesn't look like it works in this case.